### DISCUSSION TITLE
Not possible to edit questions/answers in the commercial support

### DISCUSSION THREAD

**Discussion by sturlath** [abpframework/abp#8763] [Releases]
Not sure if this is connected to the new release or not but, trying to edit questions at [QA](https://support.abp.io/QA/Questions)

gets me an endless spinner and black "screen"
![image](https://user-images.githubusercontent.com/2386572/115866147-c96dde80-a428-11eb-99c9-06b39926dc2d.png)

with this error
![image](https://user-images.githubusercontent.com/2386572/115865951-8a3f8d80-a428-11eb-8d28-8e7af688f77a.png)

**Comment by realLiangshiwei** [comment]
Thanks. We have fixed the problem, will deploy support.abp.io soon.
